### PR TITLE
Page break update

### PR DESCRIPTION
**Pull request by brittany-johnson**
https://github.com/openstax/cnx-recipes/issues/861

### CODE REVIEW

File: styles/designs/common/pdf/_page-breaks.scss [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
 nav#toc > ol > .os-toc-unit > ol > .os-toc-chapter > a > .os-number,
-nav#toc > ol > .os-toc-chapter > a > .os-number {
+nav#toc > ol > .os-toc-unit > ol > .os-toc-chapter > a > .os-text,
+nav#toc > ol > .os-toc-chapter > a > .os-number,
+nav#toc > ol > .os-toc-chapter > a > .os-text,
+ {
   page-break-after: avoid;
 }
 
+//The list of module titles should not break before so that it
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_page-breaks.scss]: so that it....? :D

File: styles/designs/common/pdf/_page-breaks.scss [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
 [data-type="document-title"],
 [data-type="title"] {
   break-after: avoid;
   break-inside: avoid;
 }
 
-.learning-objectives {
+//Unfortunately the singular and pluaral form of these classes exists
+//LO's should be orphaned from their title 
+.learning-objectives,
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_page-breaks.scss]: Should we open an issue to solve this?
Also, should this read "LOs **shouldn't** be orphaned"

Comment (brittany-johnson) [reply styles/designs/common/pdf/_page-breaks.scss]: Yes! I've made a note to open an issue

**Comment by brittany-johnson** [comment]
@helenemccarron I've updated the comments. Thanks for catching that 😅

**Pull request merged**